apps/dashboard/src/components/template-store/templates/authentication/invitation.ts

```diff
@@ -0,0 +1,28 @@
+import { StepTypeEnum, WorkflowCreationSourceEnum } from '@novu/shared';
+import { WorkflowTemplate } from '../types';
+
+export const invitationTemplate: WorkflowTemplate = {
+  id: 'invitation',
+  name: 'invitation',
```

Comment:
Let's try to make all the capitalizations aligned.

```suggestion
  name: 'Team Invite',
```